optimade/server/routers/utils.py

```diff
@@ -277,6 +278,14 @@ def get_entries(
         ),
         included=included,
     )
+    if params.response_format == "json":
+        return response_object
+    elif params.response_format == "hdf5":
```

Comment:
Need to check whether hdf5 is also enabled in the `CONFIG.enabled_response_formats` too right?

Reply:
(I now see that this is done in the `handle_query_params`, but perhaps another guard is needed here so that implementations can pick and choose which bit of the reference server they use.

Reply:
I have added an extra check.